pennylane_lightning/lightning_tensor/_tensornet.py

```diff
@@ -146,23 +151,29 @@ def __init__(
         cutoff_mode: str = "abs",
         device_name="lightning.tensor",
     ):
-        self._num_wires = num_wires
-        self._max_bond_dim = max_bond_dim
-        self._method = method
-        self._cutoff = cutoff
-        self._cutoff_mode = cutoff_mode
-        self._c_dtype = c_dtype
-
         if device_name != "lightning.tensor":
             raise DeviceError(f'The device name "{device_name}" is not a valid option.')
 
         if num_wires < 2:
             raise ValueError("Number of wires must be greater than 1.")
 
+        self._num_wires = num_wires
+        self._method = method
+        self._c_dtype = c_dtype
+        self._max_bond_dim = max_bond_dim
+        self._cutoff = cutoff
+        self._cutoff_mode = cutoff_mode
+        self._device_name = device_name
+
         self._wires = Wires(range(num_wires))
 
-        self._device_name = device_name
-        self._tensornet = self._tensornet_dtype()(self._num_wires, self._max_bond_dim)
+        self._tensornet = None
+        if self._method == "mps":
+            self._tensornet = self._tensornet_dtype()(self._num_wires, self._max_bond_dim)
+        elif self._method == "tn":
+            self._tensornet = self._tensornet_dtype()(self._num_wires)
```

Comment:
we follow the name convention from `default.tensor`

@multiphaseCFD suggestion:
> note that default.tensor use tn instead of exact. For more details, please check it [here](https://docs.pennylane.ai/en/stable/_modules/pennylane/devices/default_tensor.html#DefaultTensor). If I'm not mistaken, LT and DT should be aligned with the UI.